### PR TITLE
feat: add error-handling

### LINKED ISSUE #99: FeedAHelp: Create customs ERROR page
There are many different types of customer error pages that you can have. Here are a few examples:

- [ ] 404 Page: This is the page that is displayed when a user tries to access a page that does not exist. The 404 page should be clear and concise, and it should provide the user with a way to get back to the home page or to search for the content they are looking for.
- [ ] 500 Internal Server Error: This page is displayed when there is an error on the server that is preventing the user from accessing the content they want. The 500 error page should be informative, but it should not be too technical. The user should be able to understand what happened, but they should not be overwhelmed with technical details.
- [ ] 403 Forbidden: This page is displayed when the user does not have permission to access the content they are trying to reach. The 403 error page should be clear and concise, and it should explain why the user does not have permission to access the content.
- [ ] 401 Unauthorized: This page is displayed when the user is not logged in, and they need to log in to access the content they are trying to reach. The 401 error page should provide a link to the login page, so that the user can log in and access the content.
- [ ] 503 Service Unavailable: This page is displayed when the server is temporarily unavailable. The 503 error page should explain that the server is temporarily unavailable, and it should provide an estimated time of when the server will be back up and running.

### PR DESCRIPTION
**Pull request by Rakibs511**
# Description

Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #99 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Firmware version:
* Hardware:
* Toolchain:
* SDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: apps/feedahelp/src/components/ui/features/errors/ServerError.test.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    const tryAgainButton = getByText("Try again");
+
+    expect(title).toBeInTheDocument();
+    expect(message).toBeInTheDocument();
+    expect(image).toBeInTheDocument();
+    expect(backButton).toBeInTheDocument();
+    expect(tryAgainButton).toBeInTheDocument();
+  });
+
+//   test('clicking Try again button triggers reload', () => {
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/errors/ServerError.test.tsx]: never keep commneted shit, delete

File: apps/feedahelp/src/components/ui/features/errors/ServerError.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    }
+  return (
+    <div className='h-screen w-screen flex justify-center items-center text-center flex-col'>
+        <h1 className='text-3xl text-slate-600 md:text-6xl duration-500'>500 Internal Server Error</h1>
+
+        <p className='text-xs md:text-2xl text-slate-600 mb-10'>There was an error, please try again later.</p>
+
+        <Image src={'/errors/serverError.gif'} height={500} width={500} alt='notFound-gif'/>
+        
+        {/* back-button and try again-button */}
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/errors/ServerError.tsx]: delete commneted shit

File: apps/feedahelp/src/components/ui/features/errors/ServiceUnavailable.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+const ServiceUnavailable = ({estimatedTime}:{estimatedTime:number}) => {
+
+    const Completions = () => <span>You are good to go!</span>;
+
+    const renderer = ({ days ,hours, minutes, seconds, completed }:any) => {
+        // Render a completed state
+        if (completed) return <Completions />;
+
+          // Render a countdown
+        //   return <span>{days}:{hours}:{minutes}:{seconds}</span>;
```
Comment (TanvirAlam) [inline apps/feedahelp/src/components/ui/features/errors/ServiceUnavailable.tsx]: delete

File: apps/feedahelp/src/pages/_error.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+};
+  
+type ErrorWithStatusCodeAndTime = {
+    statusCode: 503;
+    estimatedTime: number;
+};
+  
+type ErrorProps = ErrorWithStatusCode | ErrorWithStatusCodeAndTime 
+
+// Define the Error component using the NextPage interface and ErrorProps
```
Comment (TanvirAlam) [inline apps/feedahelp/src/pages/_error.tsx]: delete

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
LGTM so far

**Review by TanvirAlam — APPROVED** [review]
LGTML so far, we will take care of these later

**Review by urmirahman — APPROVED** [review]
LGTM

**Pull request merged**